airflow/cli/commands/remote_commands/dag_command.py

```diff
@@ -537,7 +537,20 @@ def dag_test(args, dag: DAG | None = None, session: Session = NEW_SESSION) -> No
 @provide_session
 def dag_reserialize(args, session: Session = NEW_SESSION) -> None:
     """Serialize a DAG instance."""
-    # TODO: AIP-66 bundle centric reserialize
-    raise NotImplementedError(
-        "AIP-66: This command is not implemented yet - use `dag-processor --num-runs 1` in the meantime."
-    )
+    from airflow.dag_processing.bundles.manager import DagBundlesManager
+
+    manager = DagBundlesManager()
+    manager.parse_config()
+    manager.sync_bundles_to_db(session=session)
+    session.commit()
+    if args.bundle_name:
+        bundle = manager.get_bundle(args.bundle_name)
+        if not bundle:
+            raise SystemExit(f"Bundle {args.bundle_name} not found")
+        dag_bag = DagBag(bundle.path)
+        dag_bag.sync_to_db(bundle.name, bundle_version=bundle.version, session=session)
```

Comment:
```suggestion
        dag_bag.sync_to_db(bundle.name, bundle_version=bundle.get_current_version(), session=session)
```

Shouldn't this be `get_current_version`? We passed None to `get_bundle` so that we get the latest (if the bundle supports versioning of course).